desktop/apps/auction2/components/auction_grid_artwork/index.js

```diff
@@ -15,25 +16,22 @@ export default function AuctionGridArtwork({ artwork }, _) {
       <div className='auction2-grid-artwork__image-container'>
         <div className='vam-outer'>
           <div className='vam-inner'>
-            <a className='auction2-grid-artwork__image' href={artwork.href()}>
-              <img src={artwork.defaultImage().get('image_url')} alt={artwork.title}></img>
+            <a className='auction2-grid-artwork__image' href={`/artwork/${artwork._id}`}>
+              <img src={artwork.images[0].image_url} alt={artwork.title}></img>
```

Comment:
Since this is a couple jumps away (if more than one `.` away, it will crash with an undefined), it might be good to add a guard of some kind to this:

```javascript
import { get } from 'lodash'
...
const artworkImage = get(artwork, 'images.0.image_url', 'some-fallback-url')
... 
```
That said, if you're sure the image will always be there then 👍